prog/kubernetes/provision_kubernetes_node.rb

```diff
@@ -134,7 +148,32 @@ def write_hosts_file_if_needed(ip = nil)
       nap 10
     when "Failed"
       # TODO: Create a page
-      Clog.emit("JOIN CLUSTER FAILED")
+      Clog.emit("JOIN CP NODE TO CLUSTER FAILED")
+      nap 65536
+    end
+
+    nap 65536
+  end
+
+  label def join_worker
+    case vm.sshable.cmd("common/bin/daemonizer --check join_worker")
+    when "Succeeded"
+      hop_install_cni
+    when "NotStarted"
+      cp_sshable = kubernetes_cluster.cp_vms.first.sshable
```

Comment:
I do feel like I see the `cp_vms.first` formation a lot, should it have a subexpression/method?

Reply:
I made a change, but I'm not sure if this is more desirable than `cp_vms.first`.

Reply:
Resolving this for now, not a big deal to change later anyway.